underfs/swift/src/main/java/alluxio/underfs/swift/SwiftUnderFileSystem.java

```diff
@@ -131,6 +131,12 @@ public void connectFromWorker(Configuration conf, String hostname) {
   public OutputStream create(String path) throws IOException {
     LOG.debug("Create method: {}", path);
     String newPath = path.substring(Constants.HEADER_SWIFT.length());
+    if (newPath.endsWith("_SUCCESS")) {
```

Comment:
It's a bit complicated to explain, perhaps i should write a blog or something. In general when Spark saves data in object stores, it store it as
bucket/ object name 
bucket/ object name / part 1
bucket/ object name / part 2
....
bucket/ object name / _SUCCESS

All of the Hadoop drivers use this convention and so is this one https://github.com/SparkTC/stocator.
This structure is needed when object is read back by Spark.

Alluxio seems has the same structure, but i am not sure. In any case, Swift driver reduce the number of directories it creates, to achieve better performance. Therefore there is need to create this empty object and the best flow, is to create it with _SUCCESS.